waiter/src/waiter/client_tools.clj

```diff
@@ -34,6 +34,8 @@
 (def ^:const WAITER-PORT 9091)
 (def ^:const HTTP-SCHEME "http://")
 
+(def ^:const use-spnego (-> (System/getenv "USE_SPNEGO") str Boolean/parseBoolean))
```

Comment:
What happens if this environment variable is not set at all? We should support not set being the same as false